content/en/blog/_posts/2020-03-16-kubernetes-the-hard-way.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: blog
+title: SIG Scalability - Learning Kubernetes the Hard Way
+date: 2020-03-16
+slug: sig-scalability-learning-kubernetes-the-hard-way
+---
+
+**Authors:** Alex Handy
+
+There is a strain of thinking in the software development community that pushes for the most challenging and rigorous possible method of learning a new language or system. These tend to go by the moniker of "Learn \_\_ the Hard Way." Examples abound: Learn Code the Hard Way, Learn Python the Hard Way, and of course, the one that started the whole movement, Zed Shaw's courses in the topic.
```

Comment:
The phrase "strain of thinking" is interesting, but unclear. Is straining a positive or negative? Better to remove the ambiguity and opinion by stating as a data point: 

```suggestion
There is a belief in the software development community that pushes for the most challenging and rigorous possible method of learning a new language or system. These tend to go by the moniker of "Learn \_\_ the Hard Way." Examples abound: [Learn Code the Hard Way](https://shop.learncodethehardway.org/), Learn Python the Hard Way, and many others originating with Zed Shaw's courses in the topic.
```

Reply:
I believe with 3rd party links, we are limiting them at the current time (do you know if that's true, @jimangel or @zacharysarah?) But if it's okay, I'll commit in that change.

Reply:
> strain of thinking

Is that a misremembering of “train of thought” perhaps?

Reply:
Ah! Great catch, @sftim. I'm glad you don't take things like that for granite :joy: haha!

Reply:
Alternatively: “strand of thinking” is a less ambiguous rewording that is close to the original.